src/animations/backwardsCompatibleWrapper.js

```diff
@@ -0,0 +1,101 @@
+import {
+  always,
+  block,
+  call,
+  clockRunning,
+  cond,
+  set,
+  startClock,
+  stopClock,
+} from '../base';
+import { default as Clock } from '../core/AnimatedClock';
+import { default as Value } from '../core/AnimatedValue';
+
+export default function backwardsCompatibleWrapper(node, AnimationClass) {
+  return (clock, state, config) => {
+    if (config !== undefined) {
+      return node(clock, state, config);
+    }
+    const _value = clock;
+    const _config = state;
+    let returnMethod;
+    const newValue = new Value(0);
+    const newClock = new Clock();
+    const _state = AnimationClass.getDefaultState();
+    _state.position = newValue;
+
+    let finished = 0;
+
+    let enabledDetaching = true;
+    let ps = AnimationClass.persistenceStateValue;
+    let cachedValue = 0;
+    let initialized = new Value(0);
+    const initPersCache = () =>
+      cond(initialized, 0, [
+        set(_state[ps], cachedValue),
+        set(_state.finished, finished),
+        set(initialized, 1),
+      ]);
+    const wrappedNode = node(newClock, _state, _config);
+    const shouldStopClockInNexFrame = new Value(0); // needed for seq
+    const createSetNode = () =>
+      set(
+        _value,
+        block([
+          ps && [
+            initPersCache(),
+            call([_state[ps]], p => (cachedValue = p[0])),
+          ],
+          cond(shouldStopClockInNexFrame, [
+            set(shouldStopClockInNexFrame, 0),
+            stopClock(newClock),
+          ]),
+          cond(clockRunning(newClock), 0, [
+            set(newValue, _value),
+            startClock(newClock),
+          ]),
+          wrappedNode,
+          cond(_state.finished, [
+            call([], () => {
+              finished = 1;
+              if (enabledDetaching) {
+                alwaysNode.__removeChild(_value);
+              }
+              returnMethod && returnMethod({ finished: true });
+            }),
+            set(shouldStopClockInNexFrame, 1),
```

Comment:
why can't wee simply stop the clock here instead of scheduling it for the next frame?

Reply:
It's connected with sequencing. 
Take a look: if we stop clock now, `cond` in sequnce.js will not be evaluated after setting state to `finished` and it will won't trigger next animation.